CRM/Case/Form/Search.php

```diff
@@ -174,15 +174,13 @@ public function buildQuickForm() {
         $this->addRowSelectors($rows);
       }
 
-      $permission = CRM_Core_Permission::getPermission();
-
-      $tasks = CRM_Case_Task::permissionedTaskTitles($permission);
+      $tasks = CRM_Case_Task::permissionedTaskTitles(CRM_Core_Permission::getPermission());
 
       if (!empty($this->_formValues['case_deleted'])) {
-        unset($tasks[1]);
+        unset($tasks[CRM_Case_Task::DELETE]);
```

Comment:
@mattwire @eileenmcnaughton change here cause regression as it should be TASK_DELETE

Steps to replicate:
1. Go to Find Cases
2. Try to search deleted cases 
```
Error: Undefined class constant 'DELETE' in CRM_Case_Form_Search->buildQuickForm() (line 180 of /Users/monish/src/civicrm/CRM/Case/Form/Search.php).
```

Reply:
@monishdeb is that tracked somewhere?

(ie. a) good point thanks for letting us know but b) is fixing this under control ?)